routes/oauth.js

```diff
@@ -58,10 +58,14 @@ router.get("/redirect", jsonParser, async (req, res) => {
       },
     });
     const email = JSON.parse(data.body).email;
-    const user = await models.User.create({ email: email });
+    const user = await models.Subscriber.create({ email: email });
     user.saveSha1();
 
-    res.send(`Registered ${email} for breach alerts. You may now close this window/tab.`);
+    res.render("confirm", {
+      title: "Firefox Breach Alerts: Subscribed",
+      email: email,
+      user: user,
+    });
```

Comment:
The page now renders a template instead of showing a message. We should probably change the URL from `/redirect` to `/confirm` ?

Reply:
That does make sense, but it also makes sense to keep it "redirect" to stay idiomatic with the OAuth API. I'll let you make the call on this.

Reply:
In my understanding of OAuth the "redirect" is from the auth provider - Fx Accounts - to the relying party - Breach Alerts. So when I think "redirect URL" I think "to what Breach Alerts URL should FxA redirect when auth is confirmed?"

In that understanding/thinking, a URL like `/confirm` or `/auth-confirmed` makes idiomatic sense too. It would also make the URL match the contents of the page.

To update it, we'll need to change the registered redirect URL in the FxA dashboard. So, I'll file a follow-up issue to do this later.